app/controllers/api/generic_object_definitions_controller.rb

```diff
@@ -114,6 +120,10 @@ def build_generic_object_definition_options
 
     private
 
+    def generic_objects_request
```

Comment:
not quite sure what this does. why is it needed ?

Reply:
I know, it's a bit funky, but basically this controller needs to have the `before_action` hooks to allow for the subcollection to set the `associations` for requests such as `GET /api/generic_object_definitions/:id/generic_objects?associations=vms`as well as to set the hook to set `@additional_attributes`...we don't want to set the hooks unless the subject is `generic_objects`, so this method is the `if` condition on the hook. updated the name to `is_generic_objects_request`